lib/orbit-common/cache/observables/records-observable.js

```diff
@@ -0,0 +1,36 @@
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/filter';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/mergeMap';
+import 'rxjs/add/operator/concatAll';
+import PatternMatcher from 'orbit/lib/pattern-matcher';
+
+
+export default class RecordsObservable extends Observable {
+  constructor(subscribe, cache) {
+    super(subscribe);
+    this.cache = cache;
+  }
+
+  lift(operator) {
+    const observable = new RecordsObservable();
+    observable.source = this;
+    observable.operator = operator;
+    observable.cache = this.cache;
+    return observable;
+  }
+
+  patches() {
```

Comment:
`this.mergeMap` should be replaced with `this.map` (need to confirm is required for the `.last().concatAll()`)